internal/impl/babysitter.go

```diff
@@ -125,23 +118,42 @@ func NewBabysitter(ctx context.Context, app *protos.AppConfig, config *Babysitte
 }
 
 func (b *babysitter) Serve() error {
-	// Run an HTTP server that exports metrics.
-	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", prometheusPort))
-	if err != nil {
-		return fmt.Errorf("Babysitter.Serve: listen on port %d: %w", prometheusPort, err)
+	group, ctx := errgroup.WithContext(b.ctx)
+
+	if b.opts.HandleMetrics == nil {
```

Comment:
Logs: we print them with pretty printer if there is no custom way to manipulate them
Traces: we don't do anything if no custom way to manipulate them
Metrics: if no custom way to manipulate them, we export them to an http server that we will be running (in prometheus format)

Could we:
1) Create an examples or plugins directory under weaver-kube and add the jaeger example? 
2) Also add an example for metrics integration with Prometheus in the same directory? The http server code that is here can go there. I think we shouldn't do anything by default (same as for traces).

Reply:
Changed to not export metrics by default. I'll add examples in a future PR!